build/ci/release.yml

```diff
@@ -153,6 +153,7 @@ functions:
           - revision
           - created_at
           - TOOL_NAME
+          - unstable
```

Comment:
we used to pass `-unstable` but we will pass `unstable` now. Is this expected? AmI missing something?

Reply:
here we are passing the expansion not the value of the variable. Resolving.